### PR TITLE
Tidy server.py + better script loading

### PR DESCRIPTION
**Pull request by swalladge**
- tidies up top level code in `server.py` so that importing it doesn't have side effects (except still for epollreactor installing for twisted - maybe something who knows twisted can fix that sometime)
- cleaner importing scripts using `imp` - no manipulating `sys.path` anymore!
- the original changes to `sys.path` was allowing the scripts to still work with importing files directly from `feature_server` like `import commands` - now you need to `from piqueserver import commands` or `import piqueserver.commands` from scripts.

### CODE REVIEW

File: feature_server/config/scripts/afk.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 from operator import attrgetter
 from twisted.internet import reactor
 from pyspades.common import prettify_timespan
-from commands import name, get_player, add, admin
+from piqueserver.commands import name, get_player, add, admin
```
Comment (noway421) [inline feature_server/config/scripts/afk.py]: I do agree that it is The Right Way To Move Forward.

but are we keeping the backwards compatibility? I think it is important for people to be able just plug in the scripts from their existing pyspades/pysnip installation without tweaking imports

File: feature_server/server.py [hunk 2]
```diff
@@ -1048,10 +1020,13 @@ def run():
...
         script_names.append(game_mode)
 
+    script_dir = os.path.join(cfg.config_dir, 'scripts/')
     for script in script_names[:]:
         try:
-            module = __import__('scripts.%s' % script, globals(), locals(),
-                [script])
+            # NOTE: this finds and loads scripts directly from the script dir
+            # no need for messing with sys.path
+            f, filename, desc = imp.find_module(script, [script_dir])
```
Comment (noway421) [inline feature_server/server.py]: wicked 👍

Comment (swalladge) [inline feature_server/server.py]: Yeah it's pretty neat. 😄  Discovered at https://pymotw.com/2/imp/

File: feature_server/config/scripts/afk.py (hunk 1 above)
Comment (swalladge) [inline feature_server/config/scripts/afk.py]: ok, re-added to keep backwards compatibility.

**Pull request merged**